src/Pages/Home/Home.jsx

```diff
@@ -0,0 +1,13 @@
+import { useSelector } from 'react-redux';
+
+function Home() {
+  const User = useSelector((state) => state.user);
+  return (
+    <div>
+      <h1>Hello</h1>
+      {User && User.first_name}
```

Comment:
Let's use the `camelcase` approach for writing variables. In this case, the `User` variable feels like a component or a class.